en_ewt-ud-train.conllu

```diff
@@ -203002,7 +203002,7 @@
 2	service	service	NOUN	NN	Number=Sing	4	nsubj	4:nsubj	_
 3	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	cop	4:cop	_
 4	poor	poor	ADJ	JJ	Degree=Pos	0	root	0:root	SpaceAfter=No
-5	...	...	PUNCT	.	_	4	punct	4:punct	SpaceAfter=No
+5	...	...	PUNCT	.	_	4	punct	4:punct	_
```

Comment:
This SpaceAfter=No shouldn't be removed right?

Reply:
Right